src/helper/endpoint.ts

```diff
@@ -5,7 +5,8 @@ export class ZkWasmServiceEndpoint {
     constructor(
         public endpoint: string,
         public username: string,
-        public useraddress: string
+        public useraddress: string,
```

Comment:
What is this user address for? I didn't see it is using anywhere internally.
We normally should not change interface unnecessary in helper.
If you want to record user address, you can temp use user name field, as our playground also just use the address as user name